solutions/usaco-104.mdx

```diff
@@ -21,26 +21,67 @@ int main () {
 	int n, k;
 	cin >> n >> k;
 	vector<int> diff(n + 1);
-	for (int i = 0; i < k; i++) {
+	for (int i = 0; i < k; i++) { // read input and build difference array
 		int l, r;
 		cin >> l >> r;
-		l--;
+		l--; // make zero-indexed
 		r--;
 		diff[l]++;
 		diff[r + 1]--;
 	}
 
 	int sol[n];
 	int tot = 0;
-	for (int i = 0; i < n; i++) {
+	for (int i = 0; i < n; i++) { // build prefix sum array
 		tot += diff[i];
 		sol[i] = tot;
 	}
-	sort (sol, sol + n);
-	cout << sol[n/2] << '\n';
+	sort (sol, sol + n); // sort to get median
+	cout << sol[n/2] << '\n'; // output the median
 }
 
 ```
 
 </CPPSection>
+	
 </LanguageSection>
+<JavaSection>
```

Comment:
this should go inside the LanugageSection; something like:
```
<LanguageSection>

<CPPSection>

c++ code...

</CPPSection>

<JavaSection>

java code...

</JavaSection>

</LanguageSection>
```

Reply:
this is not resolved; move the javasection inside languagesection

Reply:
is it correct now?